### PR TITLE
fix(asset): droplist is stretching the asset page

### LINKED ISSUE #3805: Drop list is streching the asset page
**Describe the bug**

fill all the location and the asset type , then while clicking on the asset class, the whole asset page is being streched out

**To Reproduce**

https://care.coronasafe.in/facility/5092d953-6742-49b0-b1e3-025c53a89e8a/assets/new

**Expected behavior**

the drop list should not strech the whole form

**Screenshots**

image

### PR DESCRIPTION
**Pull request by cp-Coder**
# Bug Fix

## Proposed Changes

- Fixed stretching of the page during option select
- Replaced `SelectMenu` with `SelectMenuV2`

## Associated Issue

- Fixes https://github.com/coronasafe/care_fe/issues/3805

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

### CODE REVIEW

**Comment by nihal467** [comment]
@cp-Coder 
image

- [ ] once an option is selected in location, asset type & class, we can't unselect the droplist. for example in asset class, there is no way we can unselect it.

File: src/Components/Facility/AssetCreate.tsx [hunk 1]
```diff
@@ -480,40 +480,42 @@ const AssetCreate = (props: AssetProps) => {
...
+                          required
                           options={[
                             {
                               title: "Select",
-                              description:
-                                "Select an Asset Type from the following",
-                              value: undefined,
+                              description: "Select the asset type",
+                              value: "NONE",
                             },
```
Comment (rithviknishad) [inline src/Components/Facility/AssetCreate.tsx]: Remove the default option and use the placeholder property that's present in the new SelectMenuV2 to achieve the same instead.

File: src/Components/Facility/AssetCreate.tsx [hunk 2]
```diff
@@ -539,17 +549,26 @@ const AssetCreate = (props: AssetProps) => {
                     <div>
                       <label htmlFor="asset-class">Asset Class</label>
                       <div className="mt-2">
-                        <SelectMenu
+                        <SelectMenuV2
+                          required
```
Comment (rithviknishad) [inline src/Components/Facility/AssetCreate.tsx]: Asset class is not a required field
```suggestion
```

File: src/Components/Facility/AssetCreate.tsx [hunk 3]
```diff
@@ -539,17 +549,26 @@ const AssetCreate = (props: AssetProps) => {
...
                           options={[
-                            { title: "Not Applicable", value: undefined },
+                            { title: "Select", value: "NONE" },
```
Comment (rithviknishad) [inline src/Components/Facility/AssetCreate.tsx]: SelectMenuV2 has a placeholder prop, use that to achieve this. Adding a default option in the list of options makes it an actual valid value in the perspective of the SelectMenuV2 component.

**Review by rithviknishad — CHANGES_REQUESTED** [review]
Use the SelectMenuV2 component's features instead of hacks to achieve it.

**Review by rithviknishad — APPROVED** [review]

**Comment by nihal467** [comment]
test approved

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**